config/routes.rb

```diff
@@ -21,12 +21,15 @@
 
     get '/eligibility' => 'start_page#eligibility', as: :eligibility
     post '/eligibility' => 'start_page#determine_eligibility'
+    get '/not_eligible', to: 'start_page#not_eligible', as: :not_eligible
 
     get '/sign-up', to: 'sign_up#new', as: :sign_up
     post '/sign-up', to: 'sign_up#create'
+    get '/check-email/sign-up', to: 'sign_in#check_your_email', as: :check_email_sign_up
 
     get '/sign-in', to: 'sign_in#new', as: :sign_in
     post '/sign-in', to: 'sign_in#create'
+    get '/check-email/sign-in', to: 'sign_in#check_your_email', as: :check_email_sign_in
```

Comment:
Same reasoning here.

```suggestion
    get '/sign-in/check-email', to: 'sign_in#check_your_email', as: :sign_in_check_email
```

Reply:
Resolved in dafafa3